components/requests/DoctorInformationCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, HStack, Text, Divider, SimpleGrid, Link } from '@chakra-ui/react';
+import Card from '@components/internal/Card';
+
+type doctorInformationProps = {
```

Comment:
```suggestion
type DoctorInformationProps = {
```
Please use PascalCase for types!